src/core/attributeformmodelbase.cpp

```diff
@@ -210,6 +223,10 @@ void AttributeFormModelBase::updateAttributeValue( QStandardItem *item )
     QVariant attributeValue = mFeatureModel->feature().attribute( fieldIndex );
     item->setData( attributeValue, AttributeFormModel::AttributeValue );
   }
+  else if ( item->data( AttributeFormModel::ElementType ) == "relation" )
```

Comment:
string literal (very easy to do in QtCreator with Alt+Enter and pick the autofix)

Reply:
yeah